Review.md

```diff
@@ -0,0 +1,6 @@
+# Review Process and Rule Change
+
+## To Propose a Rule/Rule Change:
+To propose a new rule or a change to an existing rule, a player should first branch off of the Master branch.  After creating a new branch, submit your new rule or rule change via a pull request for feedback and review by fellow Jolliers.  If at least 5 Jollyball admins come to a consensus on the rule change and have incorporated the feedback listed in the new suggestion, it may be merged into the Master branch.
```

Comment:
Nitpicky as heck, but may as well capitalize `master` like it is, and throw it in some tick marks (`).